app/assets/stylesheets/welcome.scss

```diff
@@ -160,6 +160,29 @@ span.list-group-item.active {
   color: #e573fc;
   font-size: 10px;
 }
+
 .url-list {
   padding-bottom: 200px;
 }
+
+#flash_notice {
+  &.error {
+    border: solid 4px #900;
+          background-color: #ff6565;
+        }
+  &.notice {
+    border: solid 4px #26722D;
+          background-color: #a4e7a0;
```

Comment:
Color literals like `#a4e7a0` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 4 spaces, but was indented 10 spaces